grpc/src/grpc_agent_connection.rs

```diff
@@ -76,7 +76,7 @@ impl AgentConnection for GRPCAgentConnection {
             .ok_or_else(invalid_argument_empty)?
         {
             StateChangeRequestEnum::AgentHello(AgentHello { agent_name }) => {
-                log::info!("Received a hello from {:?}", agent_name);
+                log::trace!("Received a hello from {:?}", agent_name);
```

Comment:
AgentName has display implemented.
```suggestion
                log::trace!("Received a hello from '{}'", agent_name);
```